<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fawkes GIF Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>821</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fawkes Energy - GIF Creator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image_logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Arabic Blk</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../SoH_Simulator/logo/Fawkes_Energy_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse_for_folder">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>105</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans Medium</family>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_select_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Browse folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_importer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File Importer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_name_pattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>390</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_match_pattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Only pick files with the following pattern in the name</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="list_files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>390</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_found_files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>225</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Found the following files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_num_files">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>225</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create_gif">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>660</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans Medium</family>
     </font>
    </property>
    <property name="text">
     <string>Create GIF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputgif">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output GIF</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_output">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>580</width>
      <height>590</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Output GIF will be displayed here on generation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_opened_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>390</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sort_files">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>660</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Work Sans Medium</family>
     </font>
    </property>
    <property name="text">
     <string>Sort Files</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
